Makefile.js

```diff
@@ -882,6 +882,11 @@ target.checkGitCommit = function() {
 
     // Only check non-release messages
     if (!semver.valid(commitMsgs[0])) {
+        if (commitMsgs[0].slice(0, commitMsgs[0].indexOf("\n")).length > 72) {
```

Comment:
Well, let's be precise here: Is the message truncated destructively in the git reflog, or is it merely displayed in a truncated form in certain places?